app/controllers/user_controllers.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const logger = require('../logger'),
+  User = require('../models').Users;
+
+// Returns all users as object array, for testing purposes.
+exports.returnUsers = (req, res, next) => {
+  User.getAll().then(
+    x => {
+      res.send(x);
+    },
+    y => {}
+  );
+};
+
+/* Creates a new user based on the information sent through a POST req.
+   Returns a welcome message if successful, corresponding error message if not. */
+exports.buildUser = (req, res, next) => {
+  const newUser = req.body
+    ? {
+        firstName: req.body.firstName,
+        lastName: req.body.lastName,
+        password: req.body.password,
+        email: req.body.email
+      }
+    : {};
+  User.create(newUser).then(
+    result => {
+      res.send(`Successfully logged in. Welcome, ${result.firstName} ${result.lastName}!`);
+    },
+    e => {
```

Comment:
We use a errors middleware https://github.com/wolox-training/fj-express-js/blob/62427f6285a9e963be86db6abf081509ec90ede7/app/middlewares/errors.js#L14 and define errors here https://github.com/wolox-training/fj-express-js/blob/62427f6285a9e963be86db6abf081509ec90ede7/app/errors.js#L1
Simply pass something like `next(errors.internalError('Oops'))` and should do the trick (for internal errors)